content/en/platform/corda/5.0-beta/operating/notary-uniqueness-metrics.md

```diff
@@ -0,0 +1,75 @@
+---
+date: '2023-04-18'
+title: "Notary Uniqueness Metrics"
+menu:
+  corda-5-beta:
+    parent: corda-5-beta-operate
+    identifier: corda-5-beta-notary-uniqueness-metrics
+    weight: 2010
+section_menu: corda-5-beta
+
+---
+
+<style>
+table th:first-of-type {
+    width: 20%;
+}
+table th:nth-of-type(2) {
+    width: 25%;
+}
+table th:nth-of-type(3) {
+    width: 25%;
+}
+table th:nth-of-type(4) {
+    width: 30%;
+}
+</style>
+
+# Notary Uniqueness Metrics
+
+This section outlines the metrics that are provided in Corda 5 for notary and uniqueness checking functionality. Metrics have been added at the following levels:
+
+* Ledger uniqueness checker client service
+* Uniqueness checker
+* Backing store
+
+{{< note >}}
+The ledger uniqueness checker client service metrics are from the perspective of the notarization flow running on a notary virtual node. The uniqueness checker and backing store metrics are part of the uniqueness service which is running separately.
+{{< /note >}}
+
+## Ledger Uniqueness Checker Client Service
+
+The context exposed to the ledger uniqueness checker client service is minimal. As a result, only a single metric is captured:
+
+|Metric name|Type|Tags| Description|
+|-------------------------------------|-------|--------------|--------------------------------------------|
+| `ledger.uniqueness.client.run.time` | Timer |`result.type` |The time taken from requesting a uniqueness check to a response being received. The `result.type` tag is set to the relevant simple class name of the specific `UniquenessCheckResult` subclass.|
```

Comment:
```suggestion
| `ledger.uniqueness.client.run.time` | Timer |`result.type` | The time taken from requesting a uniqueness check to a response being received. The `result.type` tag is set to the specific type of uniqueness check result that was returned.|
```

Removed technical detail that would be of interest to developers in a design, but is likely meaningless / confusing to an operator.